Aptoide/aptoide-client-v8
# [ASV-1417] IronSource initialization fix

**What does this PR do?**

This PR aims at fixing the IronSource initialization. We stopped initializing IronSource back in T9 (start of January) and noticed it stopped providing us ads.
**Please** be careful with the ironsource application key. If you already have the key on your local gradle properties, comment it or remove it in order to test if it was correctly initialized for everyone that does not have the key (same problem that happened with vungle id).

**Database changed?**

   No

**Where should the reviewer start?**

- [x] MainActivity.java

**How should this be manually tested?**

Go to the mopub dashboard, check that we are getting impressions and attempts (requests) on ironsource, for the ad units that were put in production (the ones ending in 9630).
  **Please** be careful with the ironsource application key. If you already have the key on your local gradle properties, comment it or remove it in order to test if it was correctly initialized for everyone that does not have the key (same problem that happened with vungle id).

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1417](https://aptoide.atlassian.net/browse/ASV-1417)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 




**Code Review Checklist**
